sys-apps/texinfo/texinfo-7.0.3.recipe

```diff
@@ -9,44 +9,47 @@ SOURCE_URI="http://ftp.gnu.org/gnu/texinfo/texinfo-$portVersion.tar.gz"
 CHECKSUM_SHA256="3cc5706fb086b895e1dc2b407aade9f95a3a233ff856273e2b659b089f117683"
 PATCHES="texinfo-$portVersion.patchset"
 
-ARCHITECTURES="all !x86_gcc2"
+ARCHITECTURES="all"
```

Comment:
If it builds for gcc2 wouldn't it supersede the 6.* version?

Reply:
sorry, that was left-over from an experiment. I will fix it. (it doesn't build with gcc2)